package.json

```diff
@@ -40,45 +40,42 @@
 		"*": "prettier --ignore-unknown --write"
 	},
 	"dependencies": {
-		"@phenomnomnominal/tsquery": "^6.1.3",
-		"automutate": "^0.9.0",
-		"builtin-modules": "^4.0.0",
-		"chalk": "^5.3.0",
-		"commander": "^12.0.0",
-		"enquirer": "^2.4.1",
-		"glob": "^11.0.0",
-		"minimatch": "^10.0.0",
-		"strip-ansi": "^7.1.0",
-		"ts-api-utils": "^1.3.0"
+		"@phenomnomnominal/tsquery": "6.1.3",
+		"@types/eslint-plugin-markdown": "2.0.2",
```

Comment:
The other type packages are devDependencies, did you intend to put this here?

Reply:
Good catch :)